packages/e2e-test-utils/src/are-pre-publish-checks-enabled.js

```diff
@@ -9,5 +9,5 @@ import { wpDataSelect } from './wp-data-select';
  * @return {Promise<boolean>} Boolean which represents the state of prepublish checks.
  */
 export async function arePrePublishChecksEnabled() {
-	return wpDataSelect( 'core/editor', 'isPublishSidebarEnabled' );
+	return wpDataSelect( 'core', 'isPublishSidebarEnabled' );
```

Comment:
So this was a bug?

Reply:
It seems so, yes.. It wasn't used in our code base and that's why it fell through the cracks.